src/controllers/index.ts

```diff
@@ -1,5 +1,6 @@
-export * from './Auth.controller';
-export * from './Cart.controller';
-export * from './Product.controller';
-export * from './TwoFactorAuth.controller';
-export * from './User.controller';
+export * from "./Auth.controller";
+export * from "./Cart.controller";
+export * from "./Product.controller";
+export * from "./TwoFactorAuth.controller";
+export * from "./User.controller";
+export * from "./wish.controller";
```

Comment:
Try following the convention we were using of Capital letters on certain file naming
```suggestion
export * from "./Wish.controller";
```